plugins/eddn.py

```diff
@@ -168,6 +170,10 @@ def load_journal_replay(self) -> bool:
 
     def flush(self):
         """Flush the replay file, clearing any data currently there that is not in the replaylog list."""
+        if self.replayfile is None:
```

Comment:
This shouldn't be possible, as the only call sites are in EDDN.sendreplay(), which itself has a check for `self.replayfile` (not) being 'truthy'.

But, what I'd like isn't removing this check, but instead logging it at ERROR level, as it triggering would mean we've changed code elsewhere without thinking it through fully.

Reply:
Changed. I added this mostly to silence an annoying mypy complaint :D